pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -149,18 +149,11 @@
       %div{style: "text-align: center;"}
         %div{style: "display: inline-block; border-radius: 10px; padding: 20px; margin-bottom: 40px; background-image: url(/images/fit-500/homepage/codebreak_blank_note.png); background-size: cover; background-position: 50% 0%; transform: rotate(5deg)"}
           %div
-            %img#special-guest-stars{src: "images/homepage/codebreak_specialguests.png", style: "width: 300px; padding-bottom: 10px; padding-top: 30px; max-width: 100%"}
+            %img#special-guest-stars{src: "images/homepage/codebreak_specialguest.png", style: "width: 300px; padding-bottom: 10px; padding-top: 40px; max-width: 100%"}
           .headshots{style: "text-align: center"}
             .guest-1{style: "display: inline-block"}
-              %img{src: "/images/fill-200/homepage/mike_krieger_code_break.jpg", style: "border-radius: 50%; width: 150px; height: 150px;"}
-              %h3{style: "font-weight: 900; color:#191919; margin-top: 10px; margin-bottom: 6px"} Mike Krieger
-              %h3{style: "font-weight: 500; margin-top: 0px; font-size: 14px; color:#191919;"} Instagram Co-founder
-            &nbsp;
-            &nbsp;
-            .guest-2{style: "display: inline-block"}
-              %img{src: "/images/fill-200/homepage/alice_keeler_code_break.jpg", style: "border-radius: 50%; width: 150px; height: 150px;"}
-              %h3{style: "font-weight: 900; color:#191919; margin-top: 10px; margin-bottom: 6px"} Alice Keeler
-              %h3{style: "font-weight: 500; margin-top: 0px; font-size: 14px; color:#191919;"} Author & Educator
+              %img{src: "/images/fill-200/homepage/bill_gates_code_break.jpg", style: "border-radius: 50%; width: 250px; height: 250px;"}
```

Comment:
nit: if we can prefix all Code Break images in here with `codebreak_` it might be a bit easier to find them all later on?

Reply:
Will do in future changes.